.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> It depends on whether or not we really want to make Android Studio a prerequisite of the tutorials. 

The point of the tutorials is to give people a step-by-step guide that gets then up and running. If we need Android Studio to do that then we should include it, if we can get a user up and running without that that would be even better.